theworkoutinfo/ViewControllers/HomeViewController.swift

```diff
@@ -16,13 +16,19 @@ class HomeViewController: UIViewController {
     //MARK: - IBOutlets
     
     @IBOutlet weak var tableView: UITableView!
+    @IBOutlet var filterView: UIView!
+    @IBOutlet weak var tableViewTopSpaceConstraint: NSLayoutConstraint!
+    @IBOutlet weak var categoryTextField: UITextField!
     
     //MARK: - Vars
     
     var disposeBag = DisposeBag()
     var viewModel: HomeViewModel
     var rc = UIRefreshControl()
     var pageNumber = 1
+    var isCategorySelected = false
+    var categorySelected: ExerciseCategory? = nil
```

Comment:
this shouldn't be in the VC, the VC only allows the user to select the category and tell the vm